index.html

```diff
@@ -182,6 +182,12 @@
 
   1. If |dur| is an error, return 0; Otherwise return |dur|.
 
+  <p class="note">
+    Since <a>duration</a> is a {{DOMHighResTimeStamp}}, it usually represents a [=duration=] in
+    milliseconds. Since this is not enforcable in practice, <a>duration</a> can represent any unit
```

Comment:
It's not enforceable because it's merely a convention. If someone wants to pass seconds as `dur` and read it as seconds on the other side we can't stop them.

If we want to we can say that `dur` means milliseconds though and if someone is inconsistent with it it's their problem...